Client/Application/AppDelegate.swift

```diff
@@ -662,6 +662,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
 
     func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([Any]?) -> Void) -> Bool {
         if let url = userActivity.webpageURL {
+            
+            // Check for universal links and launch FxA Sign-in controller
+            let query = url.getQuery()
+            if query["deep_link"] != nil {
```

Comment:
What are the full URLs that Adjust is giving us? Is `deep_link` something they decided on?